include/swift-c/SyntaxParser/SwiftSyntaxParser.h

```diff
@@ -232,6 +232,64 @@ swiftparse_parse_string(swiftparse_parser_t, const char *source);
 /// declarations, etc.
 SWIFTPARSE_PUBLIC const char* swiftparse_syntax_structure_versioning_identifier(void);
 
+typedef struct {
+  /// Represents the range for the fixit.
+  swiftparse_range_t range;
+  /// Represent the text for replacement.
+  const char* text;
+} swiftparse_diagnostic_fixit_t;
+
+typedef enum {
+  swiftparser_diagnostic_severity_Error = 0,
```

Comment:
Use upper case for enumerators, as in https://github.com/apple/swift/blob/8521015407016d93201be98db378217cbee8af37/tools/SourceKit/tools/sourcekitd/include/sourcekitd/sourcekitd.h#L305

Reply:
Done